### ISSUE TITLE
模型转换的问题

### ISSUE THREAD

**Issue by yyl9510** [DefTruth/RobustVideoMatting.lite.ai.toolkit#27]
你好，我看官方提供了TensorFlow ，Pytorch，ONNX的模型文件，然后MNN也提供了用MNNConvert转模型的方法。
请问是如何转成rvm_mobilenetv3_fp32-480-640.mnn这种480*640的模型的呢，不知道是模型转换的哪个参数呢？
然后我看模型转换里面还有--fp16 可以将将conv/matmul/LSTM的float32参数保存为float16，模型将减小一半，精度基本无损。这是不是意味着可以用--fp16参数，然后得到的结果也就是16位精度的结果呢？
![image](https://user-images.githubusercontent.com/30204573/158776175-bb2b9468-652b-4090-9530-c692df1d0781.png)

**Comment by DefTruth** [comment]
是的。MNN是可以用--fp16来得到更小体积的模型文件的，而且可以直接调用，就和调用32位的模型一样，不需要做什么代码逻辑的改动。

**Comment by yyl9510** [comment]
那例如 rvm_mobilenetv3_fp32-480-640.mnn ， 这个后面的 tenser 的宽高是怎么设置的呢？

**Comment by DefTruth** [comment]
> 那例如 rvm_mobilenetv3_fp32-480-640.mnn ， 这个后面的 tenser 的宽高是怎么设置的呢？

这个比较复杂，可以去看看我写的一篇文章：  

* #19

**Comment by DefTruth** [comment]
先转静态ONNX->再转MNN

**Comment by yyl9510** [comment]
> 先转静态ONNX->再转MNN

感谢，我研究一下
